### PR TITLE
:bug: don't call text kwargs on line plot

### PR DESCRIPTION
**Pull request by ThomasParistech**
Don't feed text kwargs like ` ha='center'` to ax.plot (used when draw_lines is True)
`AttributeError: 'Line2D' object has no property 'ha'`

### CODE REVIEW

File: src/textalloc/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 )
 import numpy as np
 import time
-from typing import List, Union
+from typing import Any, Dict, List, Optional, Union
```
Comment (ckjellson) [inline src/textalloc/__init__.py:8]: Remove Optional here and below, not currently used for any other optional arguments.

**Review by ckjellson — CHANGES_REQUESTED** [review]
Thank you, was about to add the same changes, but if you want I can use this one. Code looks good, but would like it to be squashed into one commit and also adding the argument to the readme.md before merging.

File: src/textalloc/__init__.py (hunk 1 above)
Comment (ThomasParistech) [reply src/textalloc/__init__.py:8]: Ok, but it would be better to add it to the others ;)

Comment (ckjellson) [reply src/textalloc/__init__.py]: I agree, just didn't want to grow the task for you, will add it if I am in the mood for some refactoring ;)

**Pull request merged**